rust-toolchain.toml

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "1.74"
+components = ["rustfmt", "clippy", "llvm-tools"]
```

Comment:
Is it because we need an empty line at the end of the file?

Reply:
I think that is unnecessary.